src/web/handlers/FlingGestureHandler.ts

```diff
@@ -51,21 +91,32 @@ export default class FlingGestureHandler extends GestureHandler {
   }
 
   private tryEndFling(): boolean {
+    const velocityVector = new Vector().fromVelocity(
+      this.tracker,
+      this.keyPointer
+    );
+
+    // list of alignments to all activated directions
+    const alignmentList = [
+      Direction.LEFT,
+      Direction.RIGHT,
+      Direction.UP,
+      Direction.DOWN,
+    ].map(
+      (direction) =>
+        velocityVector.computeSimilarity(
+          new Vector().fromDirection(direction)
+        ) > this.minDirectionalAlignment && direction & this.direction
+    );
```

Comment:
```suggestion
    const alignmentList = Object.values(Direction).map((direction) => {
      const similarity = velocityVector.computeSimilarity(
        new Vector().fromDirection(direction)
      );

      return (
        similarity > this.minDirectionalAlignment && direction & this.direction
      );
    });
```

I would create separate array to hold directions or use `Object.values()`. Also the condition after `=>` is hard to read.

Also I've noticed that we use `Direction` from `constants.ts`, but we have a file called `Directions.ts`, we could change that if you want to :)